content/docs/1.6.0/v2-data-engine/features/automatic-offline-replica-rebuilding.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Automatic Offline Replica Rebuilding
 weight: 10
+aliases:
+- /spdk/features/automatic-offline-replica-rebuilding.md
```

Comment:
Before 1.6, there was no `features` folder, so should the alias be `/spdk/automatic-offline-replica-rebuilding.md`?

Reply:
I have updated the alias.